frontend/src/components/ChildrenBigPopupLocation/ChildrenBigPopupLocation.tsx

```diff
@@ -0,0 +1,176 @@
+import {
+  Input,
+  TextareaAutosize,
+  Typography,
+  Button,
+  Autocomplete,
+  TextField
+} from '@mui/material';
+import axios from 'axios';
+import React, { useState, useRef } from 'react';
+import { latlngType } from '../../../types';
+import { getFiltersForUser } from '../../static/mainFIlters';
+
+type Props = {
+  coordinate: latlngType;
+};
+
+const ChildrenBigPopupLocation = ({ coordinate }: Props) => {

```

Comment:
maybe you can change component name? maybe something like "NewLocation" or "CreateLocation", "AddLocation"

Reply:
Done